.github/workflows/publish-helm-chart.yaml

```diff
@@ -21,6 +23,20 @@ jobs:
         uses: azure/setup-helm@v1
         with:
           version: v3.5.0
+      - name: Set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Setup Chart Linting
+        id: lint
+        uses: helm/chart-testing-action@v2.1.0
```

Comment:
General comments for this and other workflow definition files:
1. Given all these supply chain attack I'd rather pin a version of GitHub action to a Git commit or Docker registry digest instead of (potentially mutable) tags.
2. We should check integrity of files downloaded with wget | curl or switch to Docker actions (e.g. check integrity of chart-releaser_1.1.1_linux_amd64.tar.gz)

Reply:
I considered your comments, please take a look, when you have time @danielpacak